### PR TITLE
Windows build support

### PR DESCRIPTION
**Pull request by kukuen**
Hi, I made the windows build working with MSVC.

For that I used the 0.4.9 release package from aubio and bundled WAF 2.0.20 with this patch manually applied : https://gitlab.com/ita1024/waf/-/commit/13d9c8566cb938cec12fab3cf731409366dbf8f7 (it's on the 2.1 branch and haven't been released yet)

### CODE REVIEW

**Comment by katyo** [comment]
@kukuen Looks interesting... Thank you so much

But I would like to avoid holding WAF source code in this repository.

**Comment by kukuen** [comment]
@katyo Sure ! I could fork WAF and fetch an artifact from there. Would that be ok ?

**Comment by katyo** [comment]
@kukuen, okay.

File: aubio-lib/build.rs [hunk 1]
```diff
@@ -1,7 +1,12 @@
 #[cfg(not(feature = "rustdoc"))]
 mod source {
-    pub const URL: &str = "https://github.com/katyo/{package}-rs/releases/download/{package}-{version}/{package}-{version}.tar.gz";
-    pub const VERSION: &str = "0.5.0-git1f23a23d";
+    pub const URL: &str = "https://github.com/aubio/aubio/archive/{version}.tar.gz";
+    pub const VERSION: &str = "0.4.9";
+
+    pub mod waf {
+        pub const URL: &str = "https://github.com/kukuen/waf/releases/download/2.0.20-msvc-fix/waf-light-{version}.tar.gz";
```
Comment (katyo) [inline aubio-lib/build.rs]: You can parameterize `URL` string with `{version}` and `{package}`:
```rust
pub const URL: &str = "https://github.com/kukuen/{package}/releases/download/{version}/waf-light-{version}.tar.gz";
```

**Review by katyo — CHANGES_REQUESTED** [review]
looks good to me, but it will be better to merge commits to single one before merging this PR with master.

File: aubio-lib/build.rs (hunk 1 above)
Comment (kukuen) [reply aubio-lib/build.rs]: I made the changes, it should be good now :)

**Pull request merged**